TDD Boot Camp/Part 01/Code/C#/OOPrinciples/Abstraction/Employee.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Abstraction
+{
+    public class Employee
+    {
```

Comment:
Use an appropriate naming convention for private variables. Use underscore.

Reply:
done